instructor-training-process.md

```diff
@@ -0,0 +1,154 @@
+	This format is being phased out,
+	as it took more effort from trainers
+	and had poorer follow-through rates than the intensive two-day versions.
+1.  **Homework.**
+    1.  After completing the course,
+        the trainee picks one core lesson from each carpentry she wants to teach
+        and submits a small change for that lesson.
+    1.  Changes do *not* have to be accepted in order for the trainee to move on,
+        but they do have to be properly formatted.
+    1.  Changes are submitted as GitHub pull requests for Software Carpentry
+        and by appending them to a Google Docs page for Data Carpentry
+	(because Data Carpentry doesn't teach Git,
+	and therefore doesn't want to require instructors to have to know it).
+    1.  Changes to content are discouraged,
+	since the lessons are now fairly stable.
+    1.  New exercises are also becoming problematic,
+	since we already have more than we can possibly use.
+    1.  Load on lesson maintainers has become unsustainable.
+1.  **Discussion sessions.**
+    1.  The trainee then takes part in an hour-long group discussion of that lesson
+        led by an experienced instructor.
+    1.  She is expected to have gone through the lesson *before* this session
+        so that she can ask lots of pointed questions during that hour.
+    1.  If the discussion leader feels the trainee is unprepared,
+        she may be asked to do some more work and try again.
+1.  **Final demonstration lessons.**
+    1.  The trainee then does a short demonstration lesson via screen sharing.
+    1.  If the examiner feels that she needs to do more work,
+        she will be given feedback and asked to try again.
+    1.  Trainees do not have to qualify separately for different topics within one Carpentry,
+        but *do* have to do a separate demonstration lesson for each Carpentry.
+1.  **Welcoming to the community.**
+    1.  Once the trainee has passed the demonstration lesson,
+        we generate a PDF certificate for the trainee by running a command-line script in
+        [the certification repository][cert-repo]
+	and send it to her along with instructions for the items below.
+    1.  The instructions sent to the new instructor tell her to:
+        1.  join the `discuss` and `instructors` lists
+	1.  fill in a form on AMY to tell us where she is, what she's comfortable teaching, etc., and
+	1.  send us a short biography and photo to add to [our website][team-page].
+    1.  We manually add the biography and photo to the website
+        (since they almost always require reformatting).
+
+## Resources
+
+1.  [AMY][amy] is the web application we use to track instructors and workshops.
+    Most of it is only accessible to administrators,
+    but it does have a form where instructors can supply or update information about themselves.
+1.  There's a web form on AMY that people can use to apply for instructor training
+    (currently disabled).
```

Comment:
Correct - individuals can't apply right now, but we would like to 
re-open that as soon as possible.